docs/src/misc.rst

```diff
@@ -197,9 +197,6 @@ API
     `UV_EINVAL` is returned. If `size` cannot accommodate the process title and
     terminating `NULL` character, the function returns `UV_ENOBUFS`.
 
-    .. warning::
```

Comment:
Can you replace these with `.. versionchanged::` to mention that these are now thread safe. There are examples in this file.

Reply:
Yep, my bad you mentioned that in the previous PR. Should I assume it will be 1.16.2?

Reply:
Looking over the commits since the last release, that seems to be correct for now.